src/main/java/domain/attendance/AttendanceWarning.java

```diff
@@ -0,0 +1,30 @@
+package domain.attendance;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum AttendanceWarning {
+    WEEDING("제적", 6),
+    INTERVIEW("면담", 3),
+    WARNING("경고", 2),
+    NONE("", 0),
+    ;
+
+    private final String status;
+    private final int absenceCount;
+
+    AttendanceWarning(String status, int absenceCount) {
+        this.status = status;
+        this.absenceCount = absenceCount;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    public static AttendanceWarning determineAttendanceWarning(int absenceIncludingTardyCount) {
+        Optional<AttendanceWarning> attendanceWarning = Arrays.stream(values())
+                .filter(value -> value.absenceCount <= absenceIncludingTardyCount).findAny();
+        return attendanceWarning.orElse(NONE);
```

Comment:
아래와 같이 메서드 체이닝을 적극 활용해서 바로 `orElse` 를 호출하셔도 됩니다 :)
```suggestion
        return Arrays.stream(values())
                .filter(value -> value.absenceCount <= absenceIncludingTardyCount).findAny();
                .orElse(NONE);
```